Add support for IE10, Firefox.

hjdivad opened the pull request:
BREAKING CHANGES:
- `card.then` → `card.promise.then`
- `conductor.load` does not start the sandbox.  The sandbox is started
  during
  `appendTo`
- `initializeDOM` is now required to call `appendTo` for each child card,
  for
  the above reason.  The default `initializeDOM` appends all children to
  `document.body`, in order.
- `oasisURL` must now refer to the polyglot file
  `conductor.js-0.1.0.js.html`.
  This file does not need to end in `html` but it does need to be served
  with
  an html content type.

Updates Oasis and RSVP.

This commit should also add support for IE9, although the Superbowl demo
behaves slightly differently between IE10 and IE9.  In the latter, the video
does not transition to the survey once it finishes playing: this might be a
problem with the YouTube API however.

There remains some CSS differences between IE and Chrome in the playground.

hjdivad committed:
Add support for IE10, Firefox.

BREAKING CHANGES:
  - `card.then` → `card.promise.then`
  - `conductor.load` does not start the sandbox.  The sandbox is started during
    `appendTo`
  - `initializeDOM` is now required to call `appendTo` for each child card, for
    the above reason.  The default `initializeDOM` appends all children to
    `document.body`, in order.
  - `oasisURL` must now refer to the polyglot file `conductor.js-0.1.0.js.html`.
    This file does not need to end in `html` but it does need to be served with
    an html content type.

Updates Oasis and RSVP.

This commit should also add support for IE9, although the Superbowl demo behaves
slightly differently between IE10 and IE9.  In the latter, the video does not
transition to the survey once it finishes playing: this might be a problem with
the YouTube API however.

There remains some CSS differences between IE and Chrome in the playground.

hjdivad commented:
cc @Cyril-sf

cyril-sf commented:
@hjdivad I made a few comments. I haven't reviewed Oasis and RSVP changes, that would be easier to do inside their own repo.

cyril-sf commented:
Probably need to start to pay attention to build version.

hjdivad committed:
SauceLabs tests pass.

Also added `test:ie` for easier testing of just IE10 and IE9.

hjdivad committed:
Added History.md

- bumped version in `package.json`.  We are now working on `0.2.0`, which has
  some (minor) breaking changes from `0.1.0`.
